src/main/java/step2/Calculator.java

```diff
@@ -0,0 +1,59 @@
+package step2;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class Calculator {
+
+  public int calculate(String formula) {
+    ArrayList<String> operatorsAndoperands = parseAndValidate(formula);
+    while (operatorsAndoperands.size() != 1) {
+      operatorsAndoperands = calculateOneStep(operatorsAndoperands);
+    }
+
+    return Integer.parseInt(operatorsAndoperands.get(0));
```

Comment:
카멜 케이스를 고려하여 변수명을 지어주세요~

```suggestion
      operatorsAndOperands = calculateOneStep(operatorsAndOperands);
```